config/development.json

```diff
@@ -59,7 +59,8 @@
 		"guided-tours/theme-sheet-welcome": true,
 		"help": true,
 		"help/courses": true,
-		"jetpack/api-cache": true,
+        "i18n/community-translator": false,
+        "jetpack/api-cache": true,
```

Comment:
The indentation is not right here. We should use two spaces for json files.

Reply:
Bad check in on my part. This file could probably be reverted fully since for dev we can pass the node env var to `npm start` until we want to release it.